model/Security/Vocabularies/CvssSeverityType.md

```diff
@@ -8,7 +8,23 @@ Specifies the CVSS base, temporal, threat, or environmental severity type.
 
 ## Description
 
-CvssSeverityType specifies the CVSS severity type, defined in the CVSS specifications as the textual representation of the numeric CVSS score. The severity type entries are inclusive of and applicable to enumerations found in CVSS versions [3](https://www.first.org/cvss/v3.0/specification-document#Qualitative-Severity-Rating-Scale) and [4](https://www.first.org/cvss/v4.0/specification-document#Qualitative-Severity-Rating-Scale). CvssSeverityType is a mandatory field because baseSeverity is required in the CVSS version [3.0](https://www.first.org/cvss/cvss-v3.0.json), [3.1](https://www.first.org/cvss/cvss-v3.1.json), and [4.0](https://www.first.org/cvss/cvss-v4.0.json) schemas. The field can be used to document the base, temporal, threat, or environmental severity.
+CvssSeverityType specifies the Common Vulnerability Scoring System (CVSS)
+severity type, defined in the CVSS specifications as the textual representation
+of the numeric CVSS score.
+
+The severity type entries are inclusive of and applicable to enumerations found
+in
+[Common Vulnerability Scoring System v3.0: Specification Document](https://www.first.org/cvss/v3.0/specification-document#Qualitative-Severity-Rating-Scale)
+and
+[Common Vulnerability Scoring System version 4.0: Specification Document](https://www.first.org/cvss/v4.0/specification-document#Qualitative-Severity-Rating-Scale).
+
+CvssSeverityType is a mandatory field because baseSeverity is required in the
+[CVSS 3.0 schema](https://www.first.org/cvss/cvss-v3.0.json),
+[CVSS 3.1 schema](https://www.first.org/cvss/cvss-v3.1.json), and
+[CVSS 4.0 schema](https://www.first.org/cvss/cvss-v4.0.json).
```

Comment:
Same question about each link having their own line... when this is rendered in markdown this will look odd. I think they can all go on the same line?

Reply:
Thank you.

In this case, a newline is introduced for each item in a list, to make it easier to track changes for each item in the list. See for example: https://github.com/spdx/spdx-spec/blob/development/v3.0.1/docs/index.md (in Raw).

Markdown will combine all lines in the same paragraph together.

For this particular file, it will be rendered like this in the production,
see: https://bact.github.io/spdx-spec/v3.0.1/model/Security/Vocabularies/CvssSeverityType/

<img width="604" alt="Screenshot 2024-08-07 at 11 17 43" src="https://github.com/user-attachments/assets/11e74ac9-24fe-4179-975e-b42aa7712ee3">

A newline can also be generally introduced when a line reach 80 characters limit, this is to accommodate text editor; and for the diffs, shorter lines mean the change set will be smaller and the change log will be easier to track.

For example, consider there's one character changing in a long paragraph. If it is a long one-line, the entire paragraph will be pulled into the change set. If there are a number of smaller lines, only one short line with the changing character will be included in the change set.

[This commit](https://github.com/spdx/spdx-spec/commit/0d6edf527636c60107edd4590eb95b376aa05d00) from the index.md example above can be an extreme example of having newlines vs having long paragraph in one line.